dipy/align/streamlinear.py

```diff
@@ -290,14 +290,14 @@ def __init__(self, metric=None, x0="rigid", method='L-BFGS-B',
         References
         ----------
         .. [Garyfallidis15] Garyfallidis et al. "Robust and efficient linear
-            registration of white-matter fascicles in the space of
-            streamlines", NeuroImage, 117, 124--140, 2015
+                            registration of white-matter fascicles in the space
```

Comment:
Not sure about the tab shift here. Is it really needed?

Reply:
I think you're right!